models/campaign.go

```diff
@@ -255,6 +257,12 @@ func PostCampaign(c *Campaign, uid int64) error {
 	if c.LaunchDate.IsZero() {
 		c.LaunchDate = time.Now()
 	}
+	if c.SMTPMinDelay == 0 {
+		c.SMTPMinDelay = 1 // default to 1 second
+	}
+	if c.SMTPMaxDelay == 0 {
+		c.SMTPMaxDelay = 10 // default to 10 seconds (most wont want the delay feature anyway)
```

Comment:
We shouldnt introduce a default delay (see my other comments). Most people will want to have the emails streamed asap.